lib/clients/incoming-webhook/client.js

```diff
@@ -14,6 +14,7 @@ var noop = require('lodash').noop;
  * @param {string} defaults.channel The default channel to use when sending a webhook.
  *      If no channel is specified, the one chosen when creating the webhook will be used.
  * @param {string} defaults.text The default text to use when sending a webhook.
+ * @param {string} defaults.link_names Do not link names and mention as default.
```

Comment:
Thank you for reviewing the description.
I updated it in https://github.com/slackapi/node-slack-sdk/pull/342/commits/11e401c76c79e0a6385bcf3fa00ce8f7e012c7b8